tests/basic.bats

```diff
@@ -95,6 +95,12 @@ load helpers
   cmp ${TESTDIR}/other-randomfile $yetanothernewroot/other-randomfile
   buildah delete $yetanothernewcid
 
+  newcid=$(buildah from new-image)
+  buildah commit --rm --signature-policy ${TESTSDIR}/policy.json $newcid containers-storage:remove-container-image
+  newroot=$(buildah mount $newcid)
+  [ "$status" -ne 0 ]
```

Comment:
Unless I'm mistaken, `bats(7)` says that `$status` is only updated when you use its `run` helper, so I don't think this'd be checking the exit status of the right command.  The `run` helper also captures the output of the command as `$output`, so there's no need for a subshell here.